### PR TITLE
Bump gke-operator to rc10

### PR DESCRIPTION
**Pull request by cmurphy**
https://github.com/rancher/rancher/issues/33406

### CODE REVIEW

File: charts/rancher-gke-operator-crd/rancher-gke-operator-crd/100.0.0+up1.1.1-rc10/Chart.yaml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+annotations:
```
Comment (cbron) [inline charts/rancher-gke-operator-crd/rancher-gke-operator-crd/100.0.0+up1.1.1-rc10/Chart.yaml:1]: Old charts folder needs to get deleted

Comment (cmurphy) [reply charts/rancher-gke-operator-crd/rancher-gke-operator-crd/100.0.0+up1.1.1-rc10/Chart.yaml]: fixed

**Review by cbron — DISMISSED** [review]

**Review by thedadams — APPROVED** [review]

**Comment by cmurphy** [comment]
rebased

**Review by cbron — DISMISSED** [review]

**Review by cbron — DISMISSED** [review]
I broke that last one :)

**Review by rmweir — DISMISSED** [review]
LGTM

**Comment by cmurphy** [comment]
rebased

File: index.yaml [hunk 2]
```diff
@@ -12,7 +12,7 @@ entries:
       catalog.cattle.io/release-name: fleet
     apiVersion: v2
     appVersion: 0.3.6-rc7
-    created: "2021-07-29T09:17:48.098424709-07:00"
+    created: "2021-07-30T08:54:02.599902241-07:00"
```
Comment (cbron) [inline index.yaml:15]: Weird that all of these changed

**Review by cbron — APPROVED** [review]

File: index.yaml (hunk 2 above)
Comment (cmurphy) [reply index.yaml:15]: they always all change when 'make charts' is run

**Pull request merged**